.github/workflows/create-pre-release.yml

```diff
@@ -0,0 +1,86 @@
+---
+name: "Create a pre-release"
+
+# This action will run when it is triggered manually
+on:
+  workflow_dispatch:
+    inputs:
+      tagName:
+        description: 'The tag name (e.g. 4.5.0-test-2 for prerelease during week 2)'
+        required: true
+        type: string
+env:
+  RELEASE_FILENAME: 'woocommerce-payments.zip'
+  
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create-release:
+    name: Create the pre-release
+    runs-on: ubuntu-20.04
+    env:
+      TAG_NAME: ${{ inputs.tagName }}
+      
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+#        with:
```

Comment:
Can we remove these commented lines if not using them?

Reply:
Sure, the plan is to uncomment it once the PR is good to go.
I was forced to comment this bit because the new workflows only exist on this branch, so the only way to test some of them (which reuse other new workflows) is to checkout this branch `add/4684-...`

I already added an action item for this at the bottom of the description:

> Remove the commented out code in create-pre-release.yml, release-code-freeze.yml and release-pr.yml to have the action run on the right branch instead of this branch by default

Reply:
I uncommented this part. We want the pre-release workflow to create a testing/ branch from `develop`.